mobile/src/screens/chat/chat.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { type ReactNode } from 'react';
+import LinearGradient from 'react-native-linear-gradient';
+
+import PlusIcon from '#assets/img/icons/plus.svg';
+import {
+  Card,
+  InputSearch,
+  Link,
+  ScrollView,
+  Text,
+  View,
+} from '#libs/components/components';
+import { AppColor, RootScreenName } from '#libs/enums/enums';
+import { useEffect, useNavigation, useSearch } from '#libs/hooks/hooks';
+
+import { Badge } from './components/components';
+import mockedChats from './libs/constants/data.json';
+import { styles } from './styles';
+
+const onPress = (): void => {
+  // This function is intentionally left empty for mocking purposes.
+};
```

Comment:
Move this inside component and start comment with TODO: implement ...

Reply:
Linter doesn't allow me to do it, It says -  Move arrow function 'onPress' to the outer scop. I think it is because the function is empty and doesn't depend on any variables. Can I leave it in outer scope? Maybe with a comment to move it after it is implemented?

Reply:
let's wrap it into useCallback and move inside

Reply:
Done